src/rpp/rpp/operators/fwd/combine_latest.hpp

```diff
@@ -0,0 +1,125 @@
+//                   ReactivePlusPlus library
+//
+//           Copyright Aleksey Loginov 2022 - present.
+//                             TC Wang 2022 - present.
+//  Distributed under the Boost Software License, Version 1.0.
+//     (See accompanying file LICENSE_1_0.txt or copy at
+//           https://www.boost.org/LICENSE_1_0.txt)
+//
+//  Project home: https://github.com/victimsnino/ReactivePlusPlus
+
+#pragma once
+
+#include <rpp/observables/details/member_overload.hpp>
+#include <rpp/observables/constraints.hpp>
+#include <rpp/utils/functors.hpp>
+#include <rpp/utils/function_traits.hpp>
+
+
+#include <tuple>
+
+namespace rpp::details
+{
+struct combine_latest_tag;
+}
+
+namespace rpp::details
+{
+
+template<constraint::decayed_type Type, typename TCombiner, constraint::observable ...TOtherObservable>
+struct combine_latest_impl;
+
+template<constraint::decayed_type Type, typename SpecificObservable>
+struct member_overload<Type, SpecificObservable, combine_latest_tag>
+{
+
+    /**
+     * \brief Combines latest emissions from current observable and other observables when any of them emits.
+     *
```

Comment:
This operator uses mutex to satisfy contract, so, it is better to add info about it (like in some other operators)